### PR TITLE
feat: cellguide sitemaps

### PR DESCRIPTION
**Pull request by sethfeingold**
- Cellguide endpoints added to /sitemap html page in new accordion layout
- New XML sitemap created for Cellguide endpoints and added to list of sitemaps at /sitemap.xml for crawlers

### CODE REVIEW

**Review by tihuan — APPROVED** [review]
Thanks so much for adding this, Seth 🙏 !!

I have a small PR with suggestions, otherwise LGTM!

https://github.com/chanzuckerberg/single-cell-data-portal/pull/5685#pullrequestreview-1607382389

**Comment by tihuan** [comment]
Actually @atarashansky pointed out that we can't use https://cellguide.cellxgene.dev.single-cell.czi.technology to parse the XML for cell types, since it only returns the first 1000 entries. So the server will need to first fetch the latest snapshot before making the second call to get the json file!

Latest snapshot id: `${CELLGUIDE_DATA_URL}/latest_snapshot_identifier`

e.g., https://cellguide.cellxgene.dev.single-cell.czi.technology/latest_snapshot_identifier

JSON file - URL: `${CELLGUIDE_DATA_URL}/${latest_snapshot_id}/celltype_metadata.json`

e.g., https://cellguide.cellxgene.dev.single-cell.czi.technology/1693591484/celltype_metadata.json

**Comment by atarashansky** [comment]
> Actually @atarashansky pointed out that we can't use https://cellguide.cellxgene.dev.single-cell.czi.technology to parse the XML for cell types, since it only returns the first 1000 entries. So the server will need to first fetch the latest snapshot before making the second call to get the json file!
> 
> Latest snapshot id: `${CELLGUIDE_DATA_URL}/latest_snapshot_identifier`
> 
> e.g., https://cellguide.cellxgene.dev.single-cell.czi.technology/latest_snapshot_identifier
> 
> JSON file - URL: `${CELLGUIDE_DATA_URL}/${latest_snapshot_id}/celltype_metadata.json`
> 
> e.g., https://cellguide.cellxgene.dev.single-cell.czi.technology/1693591484/celltype_metadata.json

I would also get the cellguide data url from `configs.CELLGUIDE_DATA_URL` (where `configs` is `src/configs/configs.js`) and use that in place of the hardcoded dev URL.

**Comment by sethfeingold** [comment]
Okay I think I've made the necessary changes! Let me know if there's anything else here that I've missed :) and thank you both for the feedback! 🥳  @tihuan @atarashansky

File: frontend/src/pages/cellguide-sitemap.xml/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const latestSnapshotRes = await fetch(
+    `${CELLGUIDE_DATA_URL}/latest_snapshot_identifier`
+  );
+
+  const latestSnapshotId = await latestSnapshotRes.json();
+
+  const response = await fetch(
+    `${CELLGUIDE_DATA_URL}/${latestSnapshotId}/celltype_metadata.json`
```
Comment (atarashansky) [inline frontend/src/pages/cellguide-sitemap.xml/index.tsx:14]: Can we also add tissues to the sitemap?

Tissues can be found in:
`${CELLGUIDE_DATA_URL}/${latestSnapshotId}/tissue_metadata.json`

Tissue pages can be accessed with:
`https://cellxgene.cziscience.com/cellguide/tissues/{TISSUE_ID}` (e.g. TISSUE_ID could be "UBERON_0002048")

**Comment by sethfeingold** [comment]
Tissues have now been added to both the /sitemap page as well as the cellguide xml sitemap @tihuan @atarashansky

Latest from main has been merged in as well with conflicts resolved.

File: frontend/src/pages/cellguide-sitemap.xml/index.tsx (hunk 1 above)
Comment (sethfeingold) [reply frontend/src/pages/cellguide-sitemap.xml/index.tsx:15]: Added!

File: frontend/package.json [hunk 2]
```diff
@@ -77,6 +77,7 @@
     "@types/react-table": "^7.7.14",
     "@types/react-window": "^1.8.5",
     "@types/sharp": "^0.32.0",
+    "@types/xml2js": "^0.4.12",
```
Comment (tihuan) [inline frontend/package.json:80]: We can remove this dep now I believe!

**Review by tihuan — COMMENTED** [review]
Just one comment left otherwise LGTM! Thanks so much, Seth!

**Review by atarashansky — APPROVED** [review]

File: frontend/package.json (hunk 2 above)
Comment (sethfeingold) [reply frontend/package.json:80]: Agreed!

**Comment by tihuan** [comment]
merging the PR, ignoring the expected failed e2e test!

**Pull request merged**